libkbfs/cr_chains.go

```diff
@@ -300,6 +300,14 @@ func (ccs *crChains) addOp(ptr BlockPointer, op op) error {
 }
 
 func (ccs *crChains) makeChainForOp(op op) error {
```

Comment:
That's probably wise, but this is everywhere in the code and I think it's out of scope for this PR as it will make it harder to read.